### PR TITLE
Chore/read replicas fixes 040324

### PR DESCRIPTION
**Pull request by joshenlim**
### Remove size selection
image

### Add info in SQL editor if trying to run mutation query on read replica
image

### CODE REVIEW

**Comment by MildTomato** [comment]
Screenshot 2024-03-05 at 12 12 05 AM

Could we use regular Button sizes and avoid stuffing padding in these?

File: apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx [hunk 1]
```diff
@@ -103,52 +111,70 @@ const UtilityTabResults = ({ id, isExecuting }: UtilityTabResultsProps) => {
...
-                    sqlAiDisclaimerComment +
-                    '\n\n' +
-                    format(sql, {
-                      language: 'postgresql',
-                      keywordCase: 'lower',
+
+          <div className="flex items-center gap-x-2">
+            {readReplicaError && (
+              <Button
+                className="py-2"
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx:130]: (non blocker)

File: apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx [hunk 2]
```diff
@@ -103,52 +111,70 @@ const UtilityTabResults = ({ id, isExecuting }: UtilityTabResultsProps) => {
...
+                  snap.resetResult(id)
+                }}
+              >
+                Switch to primary database
+              </Button>
+            )}
+            {!hasHipaaAddon && (
+              <Button
+                icon={
+                  <div className="scale-75">
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx:143]: (non blocker)

File: apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (joshenlim) [reply apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx:130]: its just to make it the same size as the debug sql button

**Comment by joshenlim** [comment]
> avoid stuffing padding in these?
the debug sql button is due to the icon size it seems, not padding
i added the padding on the switch to primary db button to match the size of the sql debug button

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**